sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs

```diff
@@ -0,0 +1,85 @@
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="InventoryProperties"/>. </summary>
+        public InventoryProperties()
+        {
+        }
+
+        /// <summary> Initializes a new instance of <see cref="InventoryProperties"/>. </summary>
+        /// <param name="cloudNativeType"> Gets or sets the cloud native resource type. </param>
+        /// <param name="cloudNativeResourceId"> Gets or sets the cloud native resource name. </param>
+        /// <param name="azureResourceId"> Gets or sets the mapped azure resource id. </param>
+        /// <param name="status"> Gets or sets the status of the inventory. </param>
+        /// <param name="statusDetails"> Gets or sets the status details. </param>
+        /// <param name="provisioningState"> The resource provisioning state. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal InventoryProperties(CloudNativeType? cloudNativeType, string cloudNativeResourceId, string azureResourceId, SolutionConfigurationStatus? status, string statusDetails, ResourceProvisioningState? provisioningState, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            CloudNativeType = cloudNativeType;
+            CloudNativeResourceId = cloudNativeResourceId;
+            AzureResourceId = azureResourceId;
+            Status = status;
+            StatusDetails = statusDetails;
+            ProvisioningState = provisioningState;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Gets or sets the cloud native resource type. </summary>
+        public CloudNativeType? CloudNativeType { get; set; }
+        /// <summary> Gets or sets the cloud native resource name. </summary>
+        public string CloudNativeResourceId { get; set; }
+        /// <summary> Gets or sets the mapped azure resource id. </summary>
+        public string AzureResourceId { get; set; }
```

Comment:
same

Reply:
It's arm-id, transformed to resource identifier type